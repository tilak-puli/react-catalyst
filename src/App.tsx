import "./App.css";
import { Routes, Route } from "react-router-dom";

// Fonts for Material UI components
import "@fontsource/roboto/300.css";
import "@fontsource/roboto/400.css";
import "@fontsource/roboto/500.css";
import "@fontsource/roboto/700.css";
import theme from "./style/theme";
import { ThemeProvider } from "@mui/material/styles";
import Todo from "./screens/todo/todo";
import Todo2 from "./screens/todo2/todo2";
import WithHeader from "./layout/withHeader";

function App() {
  return (
    <ThemeProvider theme={theme}>
      <Routes>
        <Route path="/" element={WithHeader(<Todo />)} />
        <Route path="/todo-2" element={<Todo2 />} />
      </Routes>
    </ThemeProvider>
  );
}

export default App;
