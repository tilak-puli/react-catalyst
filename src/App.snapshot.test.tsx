import renderer from "react-test-renderer";

import App from "./App";
import { BrowserRouter } from "react-router-dom";

describe("App", () => {
  it("renders learn react link", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <App />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
