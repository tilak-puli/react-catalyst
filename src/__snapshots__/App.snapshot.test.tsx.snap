// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders learn react link 1`] = `
<div>
  <h1>
    Header
  </h1>
  <div>
    <div>
      <h1
        className="MuiTypography-root MuiTypography-h1 css-o2w69a-MuiTypography-root"
      >
        First Todo
      </h1>
      <a
        href="/todo-2"
        onClick={[Function]}
      >
        Go to todo 2
      </a>
    </div>
  </div>
</div>
`;
